drivers/vulkan/rendering_device_driver_vulkan.h

```diff
@@ -209,6 +209,9 @@ class RenderingDeviceDriverVulkan : public RenderingDeviceDriver {
 		DataFormat rd_format = DATA_FORMAT_MAX;
 		VkImageCreateInfo vk_create_info = {};
 		VkImageViewCreateInfo vk_view_create_info = {};
+#ifdef DEBUG_ENABLED
+		bool created_from_extension = false;
+#endif
```

Comment:
I fell this should be the latest thing in the struct. Not a showstopper, though.

Reply:
Fixed.